### ISSUE TITLE
Deprecation warning with Sass 3.4.20

### ISSUE THREAD

**Issue by rabbitfang** [thoughtbot/bourbon#810]
When running Sass 3.4.20, the following warning is emitted:
```
DEPRECATION WARNING on line 37 of bower_components/bourbon/app/assets/stylesheets/css3/_radial-gradient.scss: #{} interpolation near operators will be simplified
in a future version of Sass. To preserve the current behavior, use quotes:

  unquote("#{$pos}#{$shape-size}#{$full}")
```

There is a deprecation notice in the [changelog entry](https://github.com/sass/sass/blob/3.4.20/doc-src/SASS_CHANGELOG.md) and there is a relevant GitHub issue for Sass: https://github.com/sass/sass/issues/1778

**Comment by tysongach** [comment]
@rabbitfang Thanks for the heads up! I opened a PR to fix this: #835

**Comment by tysongach** [comment]
Fix merged in at https://github.com/thoughtbot/bourbon/commit/c46c10d1a27f886340ed1cc0bb133b85d5b30144. We’ll cut a 4.2.7 release soon.

**Comment by tysongach** [comment]
We just released v4.2.7 which includes the fix for this.
